core/modules/system/system.install

```diff
@@ -67,7 +67,7 @@ function system_requirements($phase) {
     $requirements['php'] = array(
       'title' => $t('PHP'),
       'value' => $phpversion,
-      'description' => $t('The phpinfo() function has been disabled for security reasons. To see your server\'s phpinfo() information, change your PHP settings or contact your server administrator. For more information, <a href="@phpinfo">Enabling and disabling phpinfo()</a> handbook page.', array('@phpinfo' => 'http://drupal.org/node/243993')),
+      'description' => $t('The phpinfo() function has been disabled for security reasons. To see your server\'s phpinfo() information, change your PHP settings. For more information, <a href="@phpinfo">Enabling and disabling phpinfo()</a> handbook page.', array('@phpinfo' => 'http://drupal.org/node/243993')),
```

Comment:
Two minor things:

`your server\'s` Life might be easier for translators, if the whole string was wrapped in `"`, because then the escaping isn't necessary.

The documentation link points to drupal.org/node/243993 - does an equivalent exist for Backdrop? (Can't seem to find one.)

Reply:
If we wrapped in `"` then we wouldn't have to escape `server's`, but we would have to escape `<a href="@phpinfo">`. Because that's two instances instead of one, I'm guessing it's better to leave as-is.

As for documentation, if we don't have an equivalent page (which I assume we don't, then we should also leave as-is.

Reply:
Nice catch. :+1: Feel free to mark as resolved (I can't)

Reply:
>As for documentation, if we don't have an equivalent page (which I assume we don't, then we should also leave as-is.

Or, quickly whip up one and do it: https://backdropcms.org/user-guide/enabling-and-disabling-phpinfo-for-security-reasons 😉